ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/CloseQueryArgs.java

```diff
@@ -13,17 +13,18 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package io.confluent.ksql.api.server.protocol;
+package io.confluent.ksql.rest.entity;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.errorprone.annotations.Immutable;
-import io.confluent.ksql.api.server.PushQueryId;
 import java.util.Objects;
 
 /**
  * Represents the arguments to a close query request
  */
 @Immutable
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why do we need to add this? I thought the point in setting `.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)` in ApiJsonMapper was precisely so that we wouldn't have to individually set `@JsonIgnoreProperties(ignoreUnknown = true)` on each of the request and response types.

Reply:
This an old test in rest-model which tests that all the classes there have this.